src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTests.cs

```diff
@@ -11,27 +14,249 @@ public async Task GetInvitation_AsViewer_ShouldGetInvitation()
         {
             // Act
             var invitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
-                ViewerClient(TestFactory.PlantWithAccess), 
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
                 InitialInvitationId);
 
             // Assert
             Assert.IsNotNull(invitation);
             Assert.IsNotNull(invitation.RowVersion);
         }
 
-        //[TestMethod]
-        //public async Task CreateInvitation_AsPlanner_ShouldCreateInvitation()
-        //{
+        [TestMethod]
+        public async Task GetInvitationsByCommPkgNo_AsViewer_ShouldGetInvitations()
+        {
+            // Act
+            var invitations = await InvitationsControllerTestsHelper.GetInvitationsByCommPkgNoAsync(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                KnownTestData.CommPkgNo,
+                TestFactory.ProjectWithAccess);
+
+            // Assert
+            var invitation = invitations.First();
+            Assert.IsTrue(invitations.Count > 0);
+            Assert.AreEqual(KnownTestData.InvitationTitle, invitation.Title);
+            Assert.AreEqual(KnownTestData.InvitationDescription, invitation.Description);
+        }
+
+        [TestMethod]
+        public async Task CreateInvitation_AsPlanner_ShouldCreateInvitation()
+        {
+            const string Title = "InvitationTitle";
+            const string Description = "InvitationDescription";
+            const string Location = "InvitationLocation";
+
             // Act
-            //var id = await InvitationsControllerTestsHelper.CreateInvitationAsync(
-            //    PlannerClient(TestFactory.PlantWithAccess),
-            //    Guid.NewGuid().ToString(),
-            //    Guid.NewGuid().ToString(),
-            //    Guid.NewGuid().ToString(),
-            //    DisciplineType.DP);
+            var id = await InvitationsControllerTestsHelper.CreateInvitationAsync(
+                UserType.Planner,
+                TestFactory.PlantWithAccess,
+                Title,
+                Description,
+                Location,
+                DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participants,
+                _mcPkgScope,
+                null
+            );
+
+            var invitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                id);
+
+            // Assert
```

Comment:
move Assert-comment above line 65. Add Asserts for Title, Description and Location (if not, remove Title, Description and Location)